File: core/src/nl/t64/game/rpg/subjects/ProfileObserver.java
package nl.t64.game.rpg.subjects;

import nl.t64.game.rpg.ProfileManager;


public interface ProfileObserver {

    void onNotifyCreateProfile(ProfileManager profileManager);

    void onNotifySaveProfile(ProfileManager profileManager);

    void onNotifyLoadProfile(ProfileManager profileManager);

}

File: core/src/nl/t64/game/rpg/subjects/ProfileObserver.kt
package nl.t64.game.rpg.subjects

import nl.t64.game.rpg.ProfileManager


interface ProfileObserver {

    fun onNotifyCreateProfile(profileManager: ProfileManager)
    fun onNotifySaveProfile(profileManager: ProfileManager)
    fun onNotifyLoadProfile(profileManager: ProfileManager)

}
